docs/src/app/components/markdown-element.jsx

```diff
@@ -0,0 +1,44 @@
+const React = require('react');
```

Comment:
What do you think about using ES6 features for this new component?
For instance import instead of require?